xbmc/Application.cpp

```diff
@@ -4085,16 +4085,24 @@ PlayBackRet CApplication::PlayFile(const CFileItem& item, bool bRestart)
       CJobManager::GetInstance().PauseJobs();
     }
 
-    // don't hold graphicscontext here since player
-    // may wait on another thread, that requires gfx
-    CSingleExit ex(g_graphicsContext);
+    if (item.Exists()) // this is also an implicit call to CWakeOnAccess::WakeUpHost() if enabled
```

Comment:
Earlier in same function [here](https://github.com/t4-ravenbird/xbmc/blob/woa-main-menu/xbmc/Application.cpp#L3962) there is also a call to `videodb.open()` which also potentially initiates a server-wakeup while potentially holding the same lock? This has been present for some time now without any issues reported afaik